src/Core/src/Primitives/KeyboardAcceleratorModifiers.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace Microsoft.Maui
+{
+	[Flags]
+	public enum KeyboardAcceleratorModifiers
```

Comment:
The docs in the comment next to the field can become real docs.

Reply:
added. will add in all the other api docs after making all other changes to this PR